docs/ROADMAP.md

```
@@ -0,0 +1,88 @@
+# Stellar Portfolio Rebalancer - Public Roadmap
+
+*Last Updated: May 29, 2026*
+
+## Overview
+
+This roadmap outlines the planned development trajectory for the Stellar Portfolio Rebalancer. Items are organized into three timeframes to help contributors understand priorities and users know what to expect.
+
+##  Now (Current Sprint)
```

markdownlint-cli2 (0.22.1)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)